apps/web/controllers/pets/edit.rb

```diff
@@ -7,6 +7,7 @@ class Edit
     expose :pet
 
     def call(params)
+      # TODO How I can apply more security here?
       @pet = PetRepository.find(params[:id])
```

Comment:
I didn't got the idea, Can you give an example of the problem?

Your point is don't do something like:

```ruby
@pet = PetRepository.all_owned_by(current_user).find(params[:id])
```

I was thinking in something like that:


```ruby
@pet = PetRepository.owned_by(current_user, params[:id])
```
Or do you prefer to do like that? Or something else?

```ruby
@pet = PetQuery.owned_by(current_user, params[:id])
```